README.md

```diff
@@ -66,10 +66,8 @@ Parquet is a very active project, and new features are being added quickly. Here
 * Type-specific encoding
 * Hive integration (deprecated)
 * Pig integration
-* Cascading integration
```

Comment:
Since the code is still there, do you think we can just add '(deprecated)' here/

Reply:
Thank you for the comment, @shangxinli! I updated the PR.
Instead of removing the lines, I just added '(deprecated)' to parquet-cascading* and parquet-scrooge in README.md, just like parquet-hive.